api/v1/controllers/RegistrationController.js

```diff
@@ -351,6 +352,24 @@ function filterAttendees(req, res, next) {
 		});
 }
 
+function sendMailinglist(req, res, next) {
+	var listName = req.body.listName;
+	var mailList = mail.lists[listName]
+	var template = req.body.template;
+	
+	services.MailService.sendToList(mailList, template)
+		.then(function () {
+			res.body = {};
```

Comment:
Let's return the actual `toJSON` of the mailing list so that it looks like something happened.